package.json

```diff
@@ -31,7 +31,8 @@
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
     "fs-extra": "^0.26.7",
-    "npm-programmatic": "0.0.5"
+    "npm-programmatic": "^0.0.7",
+    "webpack": "^1.13.1"
```

Comment:
We shouldn't hardcode a Webpack dependency since #83 means a consumer has to install Webpack as a peer dependency.